acceptance/bundle/resource_deps/present_ingestion_definition/databricks.yml

```diff
@@ -0,0 +1,8 @@
+resources:
```

Comment:
It looks like TF fails to deploy with `all of
`ingestion_definition,ingestion_definition.0.objects` must be specified` but direct is not, why is that? and is it intended? TF error looks valid though